Resources/Prototypes/SS220/Entities/Sponsor/items.yml

```
@@ -0,0 +1,289 @@
+  - type: ItemCounter
+    count:
+      tags: [Cigarette]
+    composite: true
+    layerStates:
+    - cig1
+    - cig2
+    - cig3
+    - cig4
+    - cig5
+    - cig6
+  - type: Item
+    sprite: SS220/Objects/Sponsor/cigs/pack.rsi
+
+- type: entity
+  id: SponsorCigCarton
+  suffix: Шлёпа
+  parent: [ BoxCardboard, BaseBagOpenClose ]
+  name: блок сигарет Shlep'a
+  description: Картонная коробка, покрытая металлической плёнкой. Содержит 5 пачек.
+  components:
+  - type: Sprite
+    sprite: SS220/Objects/Sponsor/cigs/carton.rsi
+    layers:
+      - state: closed
+      - state: open
+        map: ["openLayer"]
+  - type: Item
+    sprite: SS220/Objects/Sponsor/cigs/carton.rsi
+    size: Normal
+  - type: Storage
+    grid:
+    - 0,0,5,1
+  - type: StorageFill
+    contents:
+      - id: CigPackGreen
+        amount: 5
+      - id: Lighter
+  - type: PhysicalComposition
+    materialComposition:
+      Plastic: 50
+
+- type: entity
+  parent: FoodSnackBase
+  id: SponsorFoodCandy
+  suffix: Шлёпа
+  name: конфетка
+  description: Разве она не должна быть чёрной?
+  components:
+  - type: Sprite
+    sprite: SS220/Objects/Sponsor/candy/base.rsi
+    state: icon
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 5
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 4
+        - ReagentId: Vitamin
+          Quantity: 1
+
+- type: entity
+  parent: BaseItem
+  id: SponsorFoodPackage
+  suffix: Шлёпа
+  name: конфетка
+  description: Что же там внутри?
+  components:
+  - type: Sprite
+    sprite: SS220/Objects/Sponsor/candy/package.rsi
+    state: icon
+  - type: Item
+    heldPrefix: chocolatebar
+    size: Tiny
+  - type: SpawnItemsOnUse
+    items:
+      - id: SponsorFoodCandy
+    sound:
+      path: /Audio/Effects/unwrap.ogg
+
+# Наборы
+
+- type: entity
+  parent: BaseItem
+  id: SponsorBoxBase
+  suffix: Шлёпа
+  name: подарочная коробка
+  abstract: true
+  description: Внутри... пусто?
+  components:
+  - type: Item
+    size: Tiny
+  - type: Sprite
+    sprite: SS220/Objects/Sponsor/sponsorbox.rsi
+    state: icon
+
+- type: entity
+  parent: BaseItem
+  id: SponsorBoxStorageBase
+  suffix: Шлёпа
+  name: продвинутая подарочная коробка
+  abstract: true
+  description: Внутри... пусто?
+  components:
+  - type: ContainerContainer
+    containers:
+      storagebase: !type:Container
+        ents: []
+  - type: UserInterface
+    interfaces:
+    - key: enum.StorageUiKey.Key
+      type: StorageBoundUserInterface
+  - type: Item
+    size: Huge
+  - type: UseDelay
+    delay: 0.5
+  - type: Storage
+    maxItemSize: Tiny
+    grid:
+    - 0,0,2,2
+  - type: Sprite
+    sprite: SS220/Objects/Sponsor/sponsorbox.rsi
+    state: icon
+
+- type: entity
+  parent: SponsorBoxBase
+  id: SponsorBoxGoldClown
+  name: золотой костюм клоуна # Золотой клуня, очевидно
+  description: Содержит в себе золотой костюм клоуна. Осторожно! Ваших рук не хватит для предметов в наборе!
```

```suggestion
  description: Содержит в себе золотой костюм клоуна. Осторожно! Набор предметов не поместится в ваших руках!
```
Мне кажется, что так более очевидно, что что-то не влезет в руки, чем абстрактное "ваших рук не хватит на это"
Если да, то ниже - тоже самое сделать